FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Farzad Zamani on 9/1/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - FeedImageMapper
+final class FeedImageMapper {
+	private struct Root: Decodable {
+		let items: [JsonFeedImageModel]
+	}
+
+	// MARK: - JsonFeedImageModel
+	struct JsonFeedImageModel: Decodable {
```

Comment:
`JsonFeedImageModel` is an implementation detail of `FeedImageMapper`, so it should be marked as `private`:
```suggestion
	private struct JsonFeedImageModel: Decodable {
```